addons/packages/harbor/2.5.3/bundle/config/globals.star

```diff
@@ -71,5 +71,9 @@ def get_resource_names_for_role(names):
   end
 end
 
+def setClusterRoleName():
```

Comment:
the implementation of this function doesn't actually `set` any value but return a value. Maybe it should be `getXXX()`?

Reply:
I actually using get at the beginning... How about we use `assembleClusterRoleName` or `concatClusterRoleName`.
@heww